docs/clean.md

```diff
@@ -3,7 +3,9 @@ The `cleanDirs` task deletes server specific files including:
 * Applications in the `apps` folder
 * Applications in the `dropins` folder
 * Server log files  
-* Server workarea files
+* Server workarea files  
+  
+Note that the `cleanDirs` task is different from the `clean` task from the Java plugin. `cleanDirs` cleans the server directories whereas `clean` removes the `build` directory. If a server is running during a `clean`, some files may remain and require a second `clean`. Adding a ```clean dependsOn 'libertyStop'``` to your `build.gradle` file can help prevent this. `cleanDirs` from the Liberty plugin already does this.
```

Comment:
a little reword on the last sentence, but not sure if "runs" is the correct verb.

The Liberty `cleanDirs` task automatically runs the `libertyStop` task.